### PR TITLE
Draw state mesh

### PR DESCRIPTION
**Pull request by davidpiuva**
Using a mesh with DrawState should now be identical to the old version.

### CODE REVIEW

**Comment by davidpiuva** [comment]
@emilwestergren Peer review?

File: source/draw/Mesh.ooc [hunk 1]
```diff
@@ -6,9 +6,7 @@
  * of the MIT license.  See the LICENSE file for details.
  */
 
-version(!gpuOff) {
```
Comment (davidpiuva) [inline source/draw/Mesh.ooc]: The version flag was preventing `GpuMesh` from being used in `DrawState`. A mesh could in theory exist for CPU rendering too even if we don't have it now.

Comment (emilwestergren) [inline source/draw/Mesh.ooc]: Yes, we want to remove as much as possible of the `Gpu` abstraction layer anyway.

File: source/draw/DrawState.ooc [hunk 2]
```diff
@@ -35,6 +37,10 @@ DrawState: cover {
 		this map = map
 		this
 	}
+	setMesh: func (mesh: Mesh) -> This {
+		this mesh = mesh
+		this
+	}
```
Comment (davidpiuva) [inline source/draw/DrawState.ooc]: This is the new interface for mesh.

**Pull request merged**